maup/repair.py

```diff
@@ -166,6 +167,23 @@ def crop_to(source, target):
 
     return cropped_geometries
 
+def expand_to(source, target):
+    """
+    Expands the source geometries to the target geometries.
+    """
+    geometries = get_geometries(source).simplify(0).buffer(0)
```

Comment:
could be worth a crs check here to ensure same crs?

Reply:
`absorb_by_shared_perimeter` will check/fail if they do not have the same crs. However, I think it's better to be explicit here -- good point!